src/pages/getting-started/app-guides/properties/index.mdx

```diff
@@ -22,7 +22,7 @@ Before you can configure the Properties building block, you need to complete the
 
 - Have an account in the <Placeholder id="DEVELOPER_CONSOLE_URL" />.
 - Create an app for which you'll create properties. If you don't know how to create an app, [here's the tutorial](/getting-started/livechat-apps/#creating-livechat-apps).
-- Add the  `properties--configuration:rw` scope in the App Authorization building block. This scope allows for creating and editing properties.
+- Add the  `properties.configuration:rw` scope in the App Authorization building block. This scope allows for creating and editing properties.
```

Comment:
Hello @bkubiak, I have included two cases I was taking care of from the #platform-docs feedback we got during the last few days. Everything connected with our case was described in the [Slack thread](https://livechat.slack.com/archives/C2XMTMQ5S/p1659953161127989). This is the only change connected with our case.